src/db/file.rs

```diff
@@ -352,4 +332,17 @@ mod test {
         let files = get_file_list(env::current_dir().unwrap().join("Cargo.toml")).unwrap();
         assert_eq!(files[0], std::path::Path::new("Cargo.toml"));
     }
+
+    #[test]
+    fn test_mime_types() {
+        assert_eq!(detect_mime(Path::new(".gitignore")), "text/plain");
+        assert_eq!(detect_mime(Path::new("hello.toml")), "text/x-toml");
+        assert_eq!(detect_mime(Path::new("hello.css")), "text/css");
+        assert_eq!(detect_mime(Path::new("hello.js")), "application/javascript");
+        assert_eq!(detect_mime(Path::new("hello.html")), "text/html");
+        assert_eq!(detect_mime(Path::new("hello.hello.md")), "text/x-markdown");
+        assert_eq!(detect_mime(Path::new("hello.markdown")), "text/markdown");
```

Comment:
Out of curiosity, do you know why `.markdown` is a different mimetype from `.md`?

Reply:
I thought it might be an official, vs unofficial kind of thing. I did more research tho, and they should both just be text/markdown. https://tools.ietf.org/html/rfc7763